usaspending_api/common/api_request_utils.py

```diff
@@ -289,35 +290,71 @@ class AutoCompleteHandler():
     @staticmethod
     # Data set to be searched for the value, and which fields to look in
     # Mode is either "contains" or "startswith"
-    def get_values_and_counts(data_set, fields, value, mode="contains"):
+    def get_values_and_counts(data_set, filter_matched_ids, pk_name):
         value_dict = {}
         count_dict = {}
 
+        for field in filter_matched_ids.keys():
+            q_args = {}
```

Comment:
This `q_args = {}; q_args[]` every time through the loop makes it look as if maybe (?) the intent is to keep building up the dict on each pass through the loop, in which case it won't work right for multiple fields b/c the dict is reset on each iteration.  I can't tell if that's the intent, though.  If I'm right, `q_args = {}` should go up above the beginning of the loop.  If I'm wrong, replacing both lines with `q_args = {pk_name + "__in": filter_matched_ids[field]}` would be slightly more concise and not look like a potential bug.